libs/Microsoft.MixedReality.WebRTC/Interop/RemoteAudioTrackInterop.cs

```diff
@@ -37,6 +52,20 @@ internal class RemoteAudioTrackInterop
             EntryPoint = "mrsRemoteAudioTrackIsOutputToDevice")]
         public static extern mrsBool RemoteAudioTrack_IsOutputToDevice(IntPtr trackHandle);
 
+        [DllImport(Utils.dllPath, CallingConvention = CallingConvention.StdCall, CharSet = CharSet.Ansi,
+            EntryPoint = "mrsRemoteAudioTrackCreateReadBuffer")]
+        public static extern uint CreateReadBuffer(IntPtr trackHandle, out ReadBufferHandle audioTrackReadBufferOut);
```

Comment:
nit: follow existing convention `RemoteAudioTrack_CreateReadBuffer`, and below `ReadBuffer_Read` and `ReadBuffer_Destroy`.

Reply:
The prefix is a bit redundant IMHO (RemoteAudioTrackInterop.RemoteAudioTrack_*) but I'll keep for consistency, can be changed later.

Reply:
Fixed